spec/std/process_spec.cr

```diff
@@ -218,7 +218,7 @@ describe Process do
     process.terminated?.should be_false
 
     # Kill, zombie now
-    process.kill
+    process.signal
```

Comment:
This is a perfect example of why it wasn't a good idea to split this patch from the POSIX abstraction PR, which also introduces .kill and .terminate. Here `.signal` is bad. This PR could also introduce them, but why bother when the other PR was perfect?

Reply:
I know, and yes i expected this :). See my comment in #8536 for the proposed way out.. But yet another split to some N other PRs and i probably lost my remaining time and energy to work on that topic..